panel/pane.py

```diff
@@ -24,6 +24,31 @@
 from .viewable import Reactive, Viewable
 
 
+def create(obj, **kwargs):
+    """
+    Creates a panel from any supplied object by wrapping it in a pane
+    and returning a corresponding Panel.
+
+    Arguments
+    ---------
+    obj: object
+       Any object to be turned into a Panel
+    **kwargs: dict
+       Any keyword arguments to be passed to the applicable Pane
+
+    Returns
+    -------
+    layout: Viewable
+       A Viewable representation of the input object
+    """
+    if isinstance(obj, PaneBase):
+        return obj
+    elif isinstance(obj, Viewable):
+        return obj
```

Comment:
Isn't PaneBase a subtype of Viewable?  So why test PaneBase explicitly?

Reply:
No reason now, just didn't tidy it up.